modules/sdk-coin-arbeth/test/fixtures/arbeth.ts

```diff
@@ -0,0 +1,156 @@
+export function getTxListRequest(address: string) {
+  return {
+    module: 'account',
+    action: 'txlist',
+    address: address,
+  };
+}
+
+export const getTxListResponse = {
+  status: '0',
+  message: 'No transactions found',
+  result: [],
+};
+
+export function getBalanceRequest(address: string) {
+  return {
+    module: 'account',
+    action: 'balance',
+    address: address,
+  };
+}
+
+export function getTokenBalanceRequest(tokenContractAddress: string, address: string) {
+  return {
+    module: 'account',
+    action: 'tokenbalance',
+    contractaddress: tokenContractAddress,
+    address: address,
+    tag: 'latest',
+  };
+}
+
+export const getBalanceResponse = {
+  status: '1',
+  message: 'OK',
+  result: '9999999999999999928',
+};
+
+export const getContractCallRequest = {
+  module: 'proxy',
+  action: 'eth_call',
+  to: '0xdf07117705a9f8dc4c2a78de66b7f1797dba9d4e',
+  data: 'a0b7967b',
+  tag: 'latest',
+};
+
+export const getContractCallResponse = {
+  jsonrpc: '2.0',
+  result: '0x0000000000000000000000000000000000000000000000000000000000002a7f',
+  id: 1,
+};
+
+export const keyShares = {
```

Comment:
Do we need these fixtures as we are using multisig now?

Reply:
I have a follow up coming up, let me remove this in the follow up